ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-inputs/index.scss

```
@@ -9,6 +9,6 @@
 
   &__separator {
     border-top: 1px solid $ui-grey;
-    margin: 24px 0 16px 0;
+    margin: 20px 0;
```

Not sure exactly what this spacing effects but would suggest adhering to `8px` grid and changing to either `24px` or `16px`
```suggestion
    margin: 16px 0;
```